src/snapshots/fuzz_crash/fuzz_crash_050.md

```diff
@@ -0,0 +1,53 @@
+# META
+~~~ini
+description=fuzz crash
+type=file
+~~~
+# SOURCE
+~~~roc
+module[]_0={
+)
+ 
+~~~
+# EXPECTED
+NIL
```

Comment:
Nice! @JRI98 This will fail tests because there's an entry under `PROBLEMS` that doesn't appear in `EXPECTED`.

You can fix that with `zig build update-expected` if you think the `PROBLEMS` entry looks fine (which I think it does in this case!)

Reply:
Hmmm I always run that command. I didn't notice that but `zig build test` is passing.

Reply:
hmm, very strange! I'll look into it, but merge in the meantime - thanks for fixing this! 😃

Reply:
@JRI98 This turned out to be a bug in the `EXPECTED` checker! Fixed in https://github.com/roc-lang/roc/pull/8049